internal/controller/exporter_controller.go

```
@@ -144,6 +151,48 @@
 	return nil
 }
 
+func (r *ExporterReconciler) reconcileStatusConditionsOnline(
+	_ context.Context,
+	exporter *jumpstarterdevv1alpha1.Exporter,
+) error {
+
+	if time.Now().Sub(exporter.Status.LastSeen.Time) > time.Minute {
```

golangci-lint (1.64.8)

S1012: should use time.Since instead of time.Now().Sub

(gosimple)